packages/core/src/transport/sendWithRetryStrategy.ts

```diff
@@ -96,10 +96,10 @@ function send(
   })
 }
 
-function retryQueuedPayloads(state: RetryState, sendStrategy: SendStrategy) {
+function retryQueuedPayloads(state: RetryState, sendStrategy: SendStrategy, reason: string) {
   const previousQueue = state.queuedPayloads
   if (previousQueue.isFull()) {
-    addTelemetryDebug('retry queue full')
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** log the intake URL used as context (to see whether this is Togs or RUM data)